yt_dlp/extractor/cda.py

```diff
@@ -199,17 +218,20 @@ def _api_extract(self, video_id):
 
     def _web_extract(self, video_id, url):
```

Comment:
We aren't using the `url` param anymore since we are constructing a new one. Then need to remove it from all method calls as well
```suggestion
    def _web_extract(self, video_id):
```